src/components/spinner/Spinner.jsx

```diff
@@ -56,7 +58,13 @@ const Spinner = ({
     className
   );
 
-  return <div {...props} className={spinnerClassNames} />;
+  return (
+    <div {...props} className={spinnerClassNames}>
+      <span className="sg-visually-hidden" aria-live="assertive" role="status">
```

Comment:
I am not sure aria-live="assertive" should be used on spinner level. This is pure visual element without app logic. Can we discuss?